packages/ra-ui-materialui/src/list/datagrid/Datagrid.tsx

```diff
@@ -346,6 +346,9 @@ export interface DatagridProps<RecordType extends RaRecord = any>
 }
 
 const injectedProps = [
+    'field',
```

Comment:
When are these fields injected? It doesn't make sense

Reply:
Because the Datagrid can be used inside a reference input